.travis.yml

```diff
@@ -22,9 +22,7 @@ before_install:
 install:
   # install a few of the dependencies that pip would otherwise try to install
   # when intalling scikit-bio
-  - travis_retry conda create -q --yes -n qiita python=3.6 pip nose flake8
-    pyzmq networkx pyparsing natsort mock future libgfortran seaborn nltk pandas
-    matplotlib scipy numpy h5py
+  - travis_retry conda create -q --yes -n qiita python=3.6 pip libgfortran numpy
```

Comment:
It's okay to remove these other dependencies? nltk and pandas are being picked up somewhere else?

Reply:
confirmed dependencies have been moved to setup.py.